stories/src/examples/DrawToolWithMarkerClusterGroup.tsx

```diff
@@ -155,21 +169,21 @@ const DrawToolWithMarkerClusterGroup: React.FC<Props> = ({
               await getMarkerGroup(layer)
               bindDistanceAndAreaToTooltip(layer)
             }}
-            onDelete={(layersInEditMode) => {
+            onDelete={(layersInEditMode: Array<ExtendedLayer>) => {
               const editLayerIds = layersInEditMode.map(({ id }) => id)
 
               // remove the markerGroups.
               if (markerGroupsRef.current) {
                 setMarkerGroups(
                   markerGroupsRef.current.filter(
-                    ({ id }) => !editLayerIds.includes(id),
+                    ({ id }: MarkerGroup) => !editLayerIds.includes(id),
```

Comment:
Can't TypeScript infer this type?

Reply:
Apparently not... At least not in my IDE